src/spinner/spinner.ts

```diff
@@ -0,0 +1,38 @@
+import { LitElement, html } from "lit";
+import { property } from "lit/decorators.js";
+import { classMap } from "lit/directives/class-map.js";
+
+export class spinner extends LitElement {
```

Comment:
Please define your class name upper case:

```suggestion
export class Spinner extends LitElement {
```